docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,10 +1,10 @@
 <!-- Thank you for submitting this PR. You are awesome!
 -->
 
-**Status**:
-- [ ] Ready
-- [ ] In Development
-- [ ] On Hold
+**Checklist**
+
+- [ ] My branch is up-to-date with the Upstream `development` branch.
+- [ ] I have added necessary documentation (if appropriate)
```

Comment:
nit: period at the end. `I have added necessary documentation (if appropriate).`